bedrock/firefox/templates/firefox/firstrun/base.html

```diff
@@ -2,12 +2,42 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
+{% add_lang_files "firefox/new/onboarding" %}
+
 {% extends "firefox/base.html" %}
 
 {% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
 
+{% if l10n_has_tag('firstrun_meta_info') %}
```

Comment:
I've added an l10n tag for the meta info here in `firstrun/base.html`

I admit, I'm a bit confused why we need this as the new default page is the only template inheriting from this file. It is possible that other firstrun templates will want to inherit from this in the future however, so I think adding these strings to a shared file might make more sense. Perhaps I'm over-thinking this though